rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -261,7 +261,7 @@ class Publisher : public PublisherBase
     if (this->can_loan_messages()) {
       // we release the ownership from the rclpp::LoanedMessage instance
       // and let the middleware clean up the memory.
-      this->do_loaned_message_publish(loaned_msg.release());
+      this->do_loaned_message_publish(loaned_msg.release().get());
```

Comment:
Would it make more sense to change the signature of this method to accept a unique_ptr? I suppose it doesn't matter, just a thought.

Reply:
OK, I'll update it and some related test source code.